Example/AblyChatExample/ContentView.swift

```diff
@@ -2,6 +2,34 @@ import Ably
 import AblyChat
 import SwiftUI
 
+// Set mode to `.live` if you wish to connect to actual instances of the Chat client in either Prod or Sandbox environments. Setting the mode to `.mock` will use the `MockChatClient`, and therefore simulate all features of the Chat app.
+private let mode = Environment.mock
```

Comment:
Maybe put it inside `Environment` as static `current`?

Reply:
Changed